draft-ietf-lamps-pq-composite-kem.md

```diff
@@ -645,8 +664,20 @@ Full specifications for the referenced algorithms can be found as follows:
 * _ML-KEM_: {{I-D.ietf-lamps-kyber-certificates}} and [FIPS.203-ipd]
 * _RSA-OAEP_: [RFC3560]
 * _X25519 / X448_: [RFC8410]
+* _HKDF_: [RFC5869]. Salt is not provided; ie the salt will be all zeros.
```

Comment:
Would it be clearer to say "i.e. the default salt is used"?  Or say "HashLen zeros" instead of "all zeros".  The salt in RFC 5869 can vary in length, and we don't want an implementation to specify a string of zeros of some other length.